### PR TITLE
add dropdown functionality in the nav menu

### PR DESCRIPTION
**Pull request by LeeRenJie**

### CODE REVIEW

File: message-me/app/javascript/packs/application.js [hunk 1]
```diff
@@ -12,3 +12,6 @@ import '@doabit/semantic-ui-sass'
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+$(document).on('turbolinks:load', function() {
```
Comment (tlkiong) [inline message-me/app/javascript/packs/application.js:15]: Can you comment this block out and take a screenshot of the page?

Comment (LeeRenJie) [reply message-me/app/javascript/packs/application.js:15]: Before
image
After
image

After adding only the dropdown works

Comment (tlkiong) [reply message-me/app/javascript/packs/application.js:15]: i ask for a screenshot of the page.
Where is it? 
Didn't I keep tell you not to assume? there are other things I want to see. Not this.

Comment (LeeRenJie) [reply message-me/app/javascript/packs/application.js:15]: When commented out
image

**Review by tlkiong — APPROVED** [review]

**Pull request merged**